src/cdk-experimental/popover-edit/lens-directives.ts

```diff
@@ -106,6 +107,18 @@ export class CdkEditControl<FormValue> implements OnDestroy, OnInit {
         break;
     }
   }
+
+  /** Triggers submit on tab out if clickOutBehavior is 'submit'. */
+  private _handleBlur(): void {
+    if (this.clickOutBehavior === 'submit') {
+      // Manually cause the form to submit before closing.
+      this._triggerFormSubmit();
+    }
+  }
+
+  private _triggerFormSubmit() {
+    this.elementRef.nativeElement!.dispatchEvent(new Event('submit'));
```

Comment:
Rather than "submitting" the form via the DOM, couldn't we call the submit handler ourselves?

Reply:
You could alternatively call` .submit()` on the form, no?

Reply:
I do need the form itself to submit in case the user added any submit handlers.